apps/catalogue/src/components/detailView/SectionField.vue

```diff
@@ -62,7 +62,7 @@ export default {
   computed: {
     fieldTypeComponentName() {
       //custom views
-      if (this.field.meta.refTable === "Linked resources") {
+      if (this.field.meta.refTableName === "Linked resources") {
```

Comment:
Should this be reworked to id, or is it fine to keep using the name here?

Reply:
I think it doesn't matter really; this is a way to customize a display, but I want double check if this is also a parameter into the client (then it MUST be a tableId)

Reply:
agreed